Update academia-hub.md (#1627)
huggingface/hub-docs: docs/hub/academia-hub.md

# Academia Hub

<Tip>
Ask your University Procurement Team to <a href="https://huggingface.co/contact/sales?from=academia" target="_blank">get in touch</a> with the university-affiliated email address to initiate the subscription process.
</Tip>

> Academia Hub is a paid offering and it provides the Hugging Face Hub’s PRO features to every student, researcher, or faculty member of an academic institution. Explore advanced tools, enhanced collaboration, and exclusive resources to accelerate your learning, research, and teaching. We would need to work with your procurement department to set that up!

Reviewer: ```suggestion
Academia Hub is a paid offering that provides the Hugging Face Hub’s PRO features to every student, researcher, or faculty member of an academic institution. Explore advanced tools, enhanced collaboration, and exclusive resources to accelerate your learning, research, and teaching. The Hugging Face team is able to work with your IT or procurement department to set the product up.
```
